.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
 
 before_script:
 - ulimit -c unlimited -S
-- pub get
+- pub get --no-precompile
```

Comment:
pub no longer precompiles by default, FYI

Reply:
great.  This will still help the stable bots though.